software/contrib/quantizer.py

```diff
@@ -238,46 +268,53 @@ def __init__(self, quantizer, n):
         self.quantizer = quantizer
         self.n = n
         self.interval_names = [
-            "-P8",
-            "-M7",
-            "-m7",
-            "-M6",
-            "-m6",
-            "-P5",
-            "-d5",
-            "-P4",
-            "-M3",
-            "-m3",
-            "-M2",
-            "-m2",
-            " P1",
-            "+m2",
-            "+M2",
-            "+m3",
-            "+M3",
-            "+P4",
-            "+d5",
-            "+P5",
-            "+m6",
-            "+M6",
-            "+m7",
-            "+M7",
-            "+P8"
+            "-Per 8",
+            "-Maj 7",
+            "-Min 7",
+            "-Maj 6",
+            "-Min 6",
+            "-Per 5",
+            "-Dim 5",
+            "-Per 4",
+            "-Maj 3",
+            "-Min 3",
+            "-Maj 2",
+            "-Min 2",
+            "Per 1",
```

Comment:
Can you add a space before this one? Otherwise it's 1 column left from everything else, due to the `+` or `-` characters.

Overall, I'm ambivalent to the `m|M` vs `Min|Maj` notation.  I think Pam's New Workout uses `mi` and `MA` for minor and major.  If it makes things more usable for people with less music theory in their background that's fine with me.

Reply:
I've just added a commit that uses the 2 character naming, I think that's much neater than what I had before but still more obvious than just single letters (and nice to keep to the same style as a common module)

I believe the way the text is displayed handles the centering regardless of the length of characters, so the space would actually push it off-centre. Does it appear off-centre on your module?

Reply:
The one-character offset probably doesn't make a huge difference. I haven't had a chance to try out the script yet, so I forgot that the layout of the UI changed too. I was thinking it was still the `{old} <- {new}` text, not a the new menu style from the photo.

Two characters + number for the intervals is probably fine.

Reply:
Accidentally commented while logged into the wrong account. Whoops!

I forgot the layout of the UI changed; I haven't actually had a chance to try it out yet.  Since the text is all in a centered row, and isn't the old `{old} <- {new}` style, having all the options occupy the same horizontal space is less of a concern.

Two letters for the interval names is probably fine.